common-arcana.lic

```diff
@@ -804,26 +803,13 @@ module DRCA
     return unless data
     return unless settings
 
+    # Skip preparing lunar spell if no moons available
+    return unless set_moon_data(data)
+
     release_cyclics if data['cyclic']
     DRC.bput('release spell', 'You let your concentration lapse', "You aren't preparing a spell") unless checkprep == 'None'
     DRC.bput('release mana', 'You release all', "You aren't harnessing any mana")
 
-    if data['moon']
-      check_moonwatch
-      moon = UserVars.moons['visible'].first
-      unless moon
-        weather ||= bput('weather', 'inside', 'You glance up at the sky.')
-        if weather =~ /inside/
-          echo "*** You're inside and there are no available moons. You're going to have a hard time casting #{data['abbrev']}"
-        end
-
-        unless moon = UserVars.moons['visible'].first
-          return
-        end
-        data['cast'] = "cast #{moon}"
-      end
-    end
```

Comment:
`set_moon_data` does these checks and updates the cast command

There is no need for the weather check, it's immaterial to any decisions being made